include/zephyr/xen/gnttab.h

```diff
@@ -41,22 +41,25 @@ int gnttab_end_access(grant_ref_t gref);
 int32_t gnttab_alloc_and_grant(void **map, bool readonly);
 
 /*
- * Provides interface to acquire free page, that can be used for
- * mapping of foreign frames. Should be freed by gnttab_put_page()
- * after usage.
+ * Provides interface to acquire one or more pages that can be used for
+ * mapping of foreign frames. Should be freed by gnttab_put_pages()
+ * after use.
+ *
+ * @param npages - number of pages to allocate.
  *
  * @return - pointer to page start address, that can be used as host_addr
  *           in struct gnttab_map_grant_ref, NULL on error.
  */
-void *gnttab_get_page(void);
+void *gnttab_get_pages(unsigned int npages);
 
 /*
- * Releases provided page, that was used for mapping foreign grant frame,
+ * Releases pages that were used for mapping foreign grant frames,
  * should be called after unmapping.
  *
- * @param page_addr - pointer to start address of used page.
+ * @param page_addr - pointer to start address of allocated buffer.
+ * @param npages - number of pages allocated for the buffer.
```

Comment:
You've changed function return type from `void` to `int`, thus you need to add `@return` here.